Documentation for checkVolume API (#380)
apache/cloudstack-documentation: source/adminguide/storage.rst

These statistics are obtained from the hypervisor directly and they represent
"current" bytes/s and IO/s values at the time of collection.

Check and repair Volume
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

When there are any leaks or any inconsistencies in the volume, then the checkVolume API can be used to

> check any such errors in the volume and helps in repairing. This feature is currently available only

Reviewer: ```suggestion
check for any such errors in the volume and helps in repairing them. This feature is currently available only
```

for KVM and volumes with QCOW2 format. This API uses "qemu-img check" command on the KVM host.

> Also a global or storage pool level setting "volume.check.and.repair.leaks.before.use" is available which allows

Reviewer: ```suggestion
Also, a global or storage pool level setting "volume.check.and.repair.leaks.before.use" is available which allows
```

> to do a check and repair leaks operation for the volume during instance start and volume attach operations.

Reviewer: ```suggestion
to check and repair any leaks of the volume during instance start and volume attach operations.
```

> This will help in repairing any leaks for the volume before using it. This is a blocking operations, meaning

Reviewer: ```suggestion
This will help in repairing any leaks of the volume before using it. This is a blocking operation, meaning
```
